lms/djangoapps/mobile_api/testutils.py

```diff
@@ -194,6 +195,26 @@ def test_non_mobile_available(self, role, should_succeed):
         self.store.update_item(self.course, self.user.id)
         self._verify_response(should_succeed, MobileAvailabilityError(), role)
 
+    # A tuple of Role Types and Boolean values that indicate whether access should be given to that role.
+    @ddt.data(
+        (auth.CourseBetaTesterRole, True),
+        (auth.CourseStaffRole, True),
+        (auth.CourseInstructorRole, True),
+        (None, True)
+    )
+    @ddt.unpack
+    @patch.dict(settings.FEATURES, {'ENABLE_MKTG_SITE': True})
+    def test_ignore_mobile_available_flag(self, role, should_succeed):
```

Comment:
Nit: Consider renaming this method to `test_non_mobile_available_ignored`.  The suggestion is for readability.  So, a developer can see that this method tests exactly the same functionality as above, but when the flag is ignored.
As a bonus, consider using DDT so the test code doesn't have to be duplicated.